test/library/characterization/test_cross_resonance_hamiltonian.py

```diff
@@ -14,17 +14,48 @@
 
 """Spectroscopy tests."""
 from test.base import QiskitExperimentsTestCase
-
+from test.extended_equality import is_equivalent
 import functools
+import io
 import numpy as np
 from ddt import ddt, data, unpack
-from qiskit import QuantumCircuit, pulse, quantum_info as qi
+from qiskit import QuantumCircuit, pulse, qpy, quantum_info as qi
 from qiskit.providers.fake_provider import FakeBogotaV2
 from qiskit.extensions.hamiltonian_gate import HamiltonianGate
 from qiskit_aer import AerSimulator
 from qiskit_experiments.library.characterization import cr_hamiltonian
 
 
+def is_equivalent_circuits(circ1: QuantumCircuit, circ2: QuantumCircuit) -> bool:
```

Comment:
Minor, but `is_equivalent_circuit` would be more natural grammatically.

Reply:
Changed as suggested.